catalog-info.yaml

```diff
@@ -23,3 +29,20 @@ spec:
   owner: group:hmpps-interventions-dev
   system: system:hmpps-interventions
   lifecycle: production
+  providesApis:
+    - api:hmpps-interventions
+
+---
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: hmpps-interventions
+  title: Intervention API
+  description: Domain API for tracking the lifecycle of CRS (Commissioned Rehabilitative Services) interventions and services
+spec:
+  type: openapi
+  lifecycle: production
+  owner: group:hmpps-interventions-dev
+  system: system:hmpps-interventions
+  definition:
+    $text: https://hmpps-interventions-service-dev.apps.live-1.cloud-platform.service.justice.gov.uk/v3/api-docs
```

Comment:
I see dependsOn: tag in the Ui file, don't we need add it here?

Reply:
No, backstage will automatically create the inverse relationship

Reply:
Ok